src/main/java/com/userservice/exception/UserExceptionHandler.java

```diff
@@ -1,25 +1,30 @@
 package com.userservice.exception;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import io.jsonwebtoken.ExpiredJwtException;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.RestControllerAdvice;
 
+@Slf4j
 @RestControllerAdvice
 public class UserExceptionHandler {
 
-    private static Logger logger = LoggerFactory.getLogger(UserExistsException.class);
-
     @ExceptionHandler(value = UserExistsException.class)
     public ResponseEntity<String> userExistsException(UserExistsException exception) {
-        logger.debug("Entering UserExceptionHandler.userExistsException with parameter exception {}.", exception);
+        log.info("Entering UserExceptionHandler.userExistsException with parameter exception {}.", exception);
+        return ResponseEntity.badRequest().body(exception.toString());
+    }
+
+    @ExceptionHandler(value = NotFoundException.class)
+    public ResponseEntity<String> notFoundException(NotFoundException exception) {
+        log.info("Entering UserExceptionHandler.notFoundException with parameter exception {}.", exception);
         return ResponseEntity.badRequest().body(exception.toString());
     }
 
-    @ExceptionHandler(value = UserNotFoundException.class)
-    public ResponseEntity<String> userNotFoundException(UserNotFoundException exception) {
-        logger.debug("Entering UserExceptionHandler.userNotFoundException with parameter exception {}.", exception);
+    @ExceptionHandler(value = ExpiredJwtException.class)
+    public ResponseEntity<String> expiredToken(ExpiredJwtException exception) {
+        log.info("Entering UserExceptionHandler.notFoundException with parameter exception {}.", exception);
```

Comment:
`UserExceptionHandler. expiredToken`

it should be unauthorized response

Reply:
Done.